packages/react/src/components/accordion/accordion-item.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import styled from 'styled-components';
+import { AccordionItemProps, AccordionBodyProps } from './accordion-types';
+import { AccordionSectionStyled, AccordionBodyStyled, HeadingStyled, ButtonStyled } from './accordion-styles';
+import { Button } from '../buttons/button';
+import { Icon } from '../icon/icon';
+import { Heading } from '../heading/heading';
+
+const AccordionSection = styled.section<AccordionBodyProps>`
```

Comment:
I think it would be easier to follow the code if the `AccordionBodyProps` interface was declared closer to it's usage.

Reply:
if I understood what you said, this comment is being kind of relate to separate style and interface.
I see that you are not fan of having kind of a separation of concerns, removing styles and types from the component itself. 
is this causing any issue? I know it is not a good practice but maybe I'm missing something here.
Could you let me know if this is causing some issues?